### DISCUSSION TITLE
Is there any way to eliminate this flicker

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1319] [Q&A]
I tried to add backgroud-color to css ,but it didnot work 

```
.nicegui-header,
.nicegui-footer {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  align-items: flex-start;
  /* gap: 1rem;
  padding: 16px; */          the size flicker can be eliminated by comment this  
  background-color: black;           the color flicker didn't disapper when I add this 
}
```

![s9sbr-rxo8e](https://github.com/zauberzeug/nicegui/assets/25096045/8476f37d-08fe-495c-afdb-d82935183143)

here is the code for the Header:
```
with ui.header().classes('items-center duration-200 p-0 no-wrap bg-[#1b1b1b]').style('box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1)'):    
        
        ui.label('LOGO').classes('mr-3 ml-4 text-h4')         
        with ui.row().classes('mr-auto'):
            for title, target in menu_items.items():
                ui.link(title, target).classes(replace='text-lg text-white')
        ui.button(on_click=menu_right.toggle, icon='menu').props('flat color=white round').classes('h=3 mr-3')
```

**Comment by falkoschindler** [comment]
What is the purpose of `duration-200`?

**Reply by v479038280** [reply]
`  background-color: black !important;               
`this works .... . 
 I'm not familar with TaiWindCSS ,some attribute may be meaningless Just try the effects

here is the result
![s9sbr-rxo8e](https://github.com/zauberzeug/nicegui/assets/25096045/8b1fae41-7ae0-446a-bfdd-9abb222e11c8)

**Reply by v479038280** [reply]
If the custom setting should overwrite  the default setting , and the default should not take effect
